crates/rspack_binding_options/src/options/raw_module/js_loader.rs

```diff
@@ -0,0 +1,294 @@
+use napi_derive::napi;
+#[cfg(feature = "node-api")]
+use {
+  napi::bindgen_prelude::*,
+  napi::NapiRaw,
+  rspack_binding_macros::call_js_function_with_napi_objects,
+  rspack_core::{Loader, LoaderContext, LoaderRunnerContext},
+  rspack_error::internal_error,
+  rspack_identifier::{Identifiable, Identifier},
+  rspack_napi_shared::threadsafe_function::{ThreadsafeFunction, ThreadsafeFunctionCallMode},
+  rspack_napi_shared::{NapiResultExt, NAPI_ENV},
+};
+
+#[napi(object)]
+pub struct JsLoader {
+  /// composed loader name, xx-loader$yy-loader$zz-loader
```

Comment:
Like the previous PR said, the composed JS loaders will now be joined with `$` for better interoperability between the Rust and Node side.